ExaltedApotheosisPreReqs.cs

```diff
@@ -0,0 +1,59 @@
+    //cs_include Scripts/CoreBots.cs
+    //cs_include Scripts/CoreFarms.cs
+    using Skua.Core.Interfaces;
+
+    public class ExaltedApotheosisPreReqs
+    {
+        public IScriptInterface Bot => IScriptInterface.Instance;
+        public CoreBots Core => CoreBots.Instance;
+        public CoreFarms Farm = new();
+
+        public void ScriptMain(IScriptInterface bot)
+        {
+            Core.SetOptions();
+
+            PreReqs();
+
+            Core.SetOptions(false);
+        }
+
+        public void PreReqs()
+        {
+            Core.AddDrop("Exalted Node", "Thaumaturgus Alpha", "Apostate Alpha", "Exalted Relic Piece", "Exalted Forgemetal", "Exalted Artillery Shard");
+            if (Core.CheckInventory("Exalted Node", 300) && Core.CheckInventory("Thaumaturgus Alpha") && Core.CheckInventory("Apostate Alpha"))
+                Core.Logger("Got all prerequisites! Kill the ultra bosses manually for insignias next to complete Exalted Apotheosis.");

```

Comment:
for better boating experience u should make a check for insignias if the user have them buy exalted apotheosis if not give him a log check this example for example: 
Darkon\DarkonDebris2ReconstructedPrerequisites.cs

Reply:
Yea I was gonna add that next